install_config/cluster_metrics.adoc

```diff
@@ -475,6 +475,78 @@ version corresponds to the very latest version available and can cause issues if
 newer version not meant to function on the version of {product-title} you are currently running.
 ====
 
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
```

Comment:
Let's explicitly identify the default for this one: https://github.com/zgalor/openshift-ansible/blob/3cf3f7de6bbbe7ac9eecf3f838b10c78da371677/roles/openshift_prometheus/defaults/main.yaml#L5

Reply:
I think this has been addressed. Please let me know if I can do so more clearly.

Reply:
I also think its covered by https://github.com/openshift/openshift-docs/pull/5591/files#diff-ca2a8ff7fac87b827ba21942e6b79bfbR902 however we might want to explicitly call out the default name?